resolvers.js

```diff
@@ -159,12 +172,33 @@ const resolvers = {
         // TODO: Double check defaults
         userId: context.getUser()._id,
         databaseId: databaseId,
-        title: 'untitled',
+        categoryId: categoryId,
+        title: title,
         blocks: []
       }).save();
 
-      databaseDocument.notes.push(newNote._id);
-      await databaseDocument.save();
+      const categoryDocument = await Category.findOne({ _id: categoryId });
+
+      const noteCopy = [...categoryDocument.notes];
+
+      if (databaseDocument.currentView === 'board') {
+        databaseDocument.notes.push(newNote._id);
+        await databaseDocument.save();
+        noteCopy.splice(index, 0, newNote._id);
+      } else {
+        databaseDocument.notes.splice(index, 0, newNote._id);
+        await databaseDocument.save();
+        noteCopy.push(newNote._id);
+      }
```

Comment:
can use an enum instead of `'board'`. also, its better to be more specific in the else block, incase someone else adds in functionality, and it goes straight into the else block